include/SDL3/SDL_process.h

```diff
@@ -0,0 +1,271 @@
+
+#include <SDL3/SDL_begin_code.h>
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct SDL_Process SDL_Process;
+
+/**
+ * Create a new process.
+ *
+ * The path to the executable is supplied in args[0]. args[1..N] are additional arguments passed on the command line of the new process, and the argument list should be terminated with a NULL, e.g.:
+ *
+ * ```c
+ * const char *args[] = { "myprogram", "argument", NULL };
+ * ```
+ *
+ * See SDL_CreateProcessWithProperties() for more details.
+ *
+ * \param args the path and arguments for the new process.
+ * \param enable_IO SDL_TRUE to enable reading and writing to the process, SDL_FALSE to have the process output go to the application's output.
+ * \returns the newly created and running process, or NULL if the process couldn't be created.
+ *
+ * \sa SDL_CreateProcessWithProperties
+ * \sa SDL_GetProcessProperties
+ * \sa SDL_ReadProcess
+ * \sa SDL_WriteProcess
+ * \sa SDL_KillProcess
+ * \sa SDL_WaitProcess
+ * \sa SDL_DestroyProcess
+ *
+ * \threadsafety It is safe to call this function from any thread.
+ *
+ * \since This function is available since SDL 3.0.0.
+ */
+extern SDL_DECLSPEC SDL_Process *SDLCALL SDL_CreateProcess(const char * const *args, SDL_bool enable_IO);
+
+/**
+ * Create a new process with the specified properties.
+ *
+ * These are the supported properties:
+ *
+ * - `SDL_PROP_PROCESS_CREATE_ARGS_POINTER`: an array of strings containing the program to run, any arguments, and a NULL pointer, e.g. const char *args[] = { "myprogram", "argument", NULL }. This is a required property.
+ * - `SDL_PROP_PROCESS_CREATE_ENVIRONMENT_POINTER`: an array of strings containing variable=value, and a NULL pointer, e.g. const char *env[] = { "PATH=/bin:/usr/bin", NULL }. If this property is set, it will be the entire environment for the process, otherwise the current environment is used.
+ * - `SDL_PROP_PROCESS_CREATE_STDIN_BOOLEAN`: true if an SDL_IOStream should be created as input to the process and available as `SDL_PROP_PROCESS_STDIN_POINTER` in the properties for the created process. By default the process has no input.
+ * - `SDL_PROP_PROCESS_CREATE_STDOUT_BOOLEAN`: true if an SDL_IOStream should be created for output from the process and available as `SDL_PROP_PROCESS_STDOUT_POINTER` in the properties for the created process. By default the process shares the output of the current application.
+ * - `SDL_PROP_PROCESS_CREATE_STDERR_BOOLEAN`: true if an SDL_IOStream should be created for error output from the process and available as `SDL_PROP_PROCESS_STDERR_POINTER` in the properties for the created process. By default the process shares the error output of the current application.
+ * - `SDL_PROP_PROCESS_CREATE_REDIRECT_STDERR_BOOLEAN`: true if the error output of the process should be redirected into the standard output of the process. This property has no effect if `SDL_PROP_PROCESS_CREATE_STDERR_BOOLEAN` is true.
```

Comment:
I'd personally call this `STDERR_TO_STDOUT`:

```suggestion
 * - `SDL_PROP_PROCESS_CREATE_STDERR_TO_STDOUT_BOOLEAN`: true if the error output of the process should be redirected into the standard output of the process. This property has no effect if `SDL_PROP_PROCESS_CREATE_STDERR_BOOLEAN` is true.
```